napari/_qt/qt_viewer.py

```diff
@@ -303,8 +277,36 @@ def __init__(
 
         for layer in self.viewer.layers:
             self._add_layer(layer)
-        for overlay in self.viewer._overlays.values():
-            self._add_overlay(overlay)
+
+    @property
+    def view(self):
+        """
+        Rectangular  vispy viewbox widget in which a subscene is rendered. Access directly within the QtViewer will
+        become deprecated.
+        """
+        warnings.warn(
+            trans._(
+                "Access to QtViewer.view will become deprecated in the napari 0.6.0. Change to QtViewer.canvas.view instead."
```

Comment:
Still not sure what this was supposed to be. @Czaki ? I think this should say "Access to QtViewer.view is deprecated since 0.6.0".

Reply:
An essential part of the message is before which version one needs to update his code. 

Maybe the proper message will be:

```
"Access to QtViewer.view is deprecated since 0.5.0 and will be removed in the napari 0.6.0. Change to QtViewer.canvas.view instead."
```

Reply:
I agree this is more neat. Changed with [7316d93](https://github.com/napari/napari/pull/5432/commits/7316d9320b09a9e11ea1b1b921104c49107701b8).